### DISCUSSION TITLE
想更新添加新资源

### DISCUSSION THREAD

**Discussion by 2003Tim** [beiyuouo/hainanu-course-comments#14] [General]
学长学姐可以开放权限嘛

**Comment by yyyanbj** [comment]
如果你熟悉 Git 操作可以提交 Pull requests，参考：https://www.bj-yan.top/hainanu-course-comments/0.%E8%B5%84%E6%BA%90%E5%AF%BC%E8%88%AA/%E4%BD%BF%E7%94%A8%E6%8C%87%E5%8D%97/

Looking forward to your contribution! :)
